### ISSUE TITLE
Is there a way to prevent drag cancel when user touches display with second finger?

### ISSUE THREAD

**Issue by Okami92** [yahoo/react-dnd-touch-backend#64]

**Comment by JoaoMosmann** [comment]
Currently, it's not possible. Every time you do a TouchStart it clears the list of dragged sources as you can see here 

https://github.com/oyshan/react-dnd-touch-backend/blob/0a89e63cd3da303c40d0bb224fc3531d9a8092ec/src/Touch.js#L239

Maybe @longlho could give a better insight.

**Comment by Okami92** [comment]
Thanks for response!

Well, that's sad to hear. This project will be running on windows desktop machine with big touch screen, and client insist that it's a problem and right now I am trying to solve it with something like this: 

`if(document.addEventListener) {
  document.addEventListener('contextmenu', function(e) { e.stopPropagation(); });
}
else if(document.attachEvent) {
  document.attachEvent('oncontextmenu', function() { event.cancelBubble = true; });
}`

As I understand, if you run application in the desktop browser with touch screen, touching it accidentally
with second finger considered as clicking on the right mouse button and as a result, context menu appears. With construction above, I managed to prevent drag cancel, but in order to proceed dragging, you need to remove first finger from screen and then tap it again (or mouseup -> mousedown in dev tools emulator). Then custom drag layer will continue to move.

Sorry for a lot of text. I am a junior developer, just desperately trying to find solution that will satisfy customer. Will appreciate all the help. Thanks in advance.

**Comment by longlho** [comment]
yeah unfortunately it's by design due to differences between native HTML5 drag & touch/mouse event (more prone to complicated scenario like this)

**Comment by JoaoMosmann** [comment]
What could be done, is to attach the drag events to the Touch Identifier, so each touch would have its own state. However, I think this could be a breaking changing.

So when you `touchend` the second touch, it wouldn't cancel the first touch dragging.
It would also allow having multiple draggings at the same time, which I'm not sure if it's a good feature.

Also, the Touch Identifier is not well supported across the browsers [Touch API](https://developer.mozilla.org/en-US/docs/Web/API/Touch). But I think it's possible to develop a way around to identify the touches.

So, if you're really in need for this feature, do a fork of the project, I can help a little in my free time.

**Comment by Okami92** [comment]
That would be great, thanks!
